tests/subsys/logging/log_switch_format/boards/intel_adsp_cavs25.conf

```diff
@@ -0,0 +1,5 @@
+CONFIG_LOG_BACKEND_UART=n
+CONFIG_LOG_BACKEND_UART_OUTPUT_SYST=n
+CONFIG_LOG_BACKEND_ADSP=y
+CONFIG_LOG_MIPI_SYST_ENABLE=y
+CONFIG_LOG_BACKEND_ADSP_OUTPUT_SYST=y
```

Comment:
I'm curious/confused why there are these cavs config files, but no mention of these platforms in the `testcase.yaml` - is this just a convenience for those who happen to be able to run twister on these platforms?